src/runner.sh

```diff
@@ -213,7 +213,11 @@ function runner::run_test() {
 
   local end_time
   end_time=$(clock::now)
-  local duration=$((end_time - start_time))
+  local duration
+  local duration_ns
+  duration_ns=$(math::calculate "($end_time - $start_time) ")
+  duration=$(math::calculate "$duration_ns / 1000000")
```

Comment:
Let's keep the local declaration as close as possible to the scope of its assignment (within the possible context) 
```suggestion
  local duration_ns
  duration_ns=$(math::calculate "($end_time - $start_time) ")
  local duration
  duration=$(math::calculate "$duration_ns / 1000000")
```